components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -386,6 +419,18 @@ func main() {
 			step:     provisioning.NewEmsBindStep(db.Operations(), cfg.Database.SecretKey),
 			disabled: cfg.Ems.Disabled,
 		},
+		{
+			weight:   7,
+			step:     provisioning.NewClsBindStep(clsConfig, clsClient, db.Operations(), cfg.Database.SecretKey),
+			disabled: cfg.Cls.Disabled,
+		},
+
+		{
+			weight:   8,
+			step:     provisioning.NewClsAuditLogOverridesStep(db.Operations(), cfg.AuditLog, cfg.Database.SecretKey),
+			disabled: !cfg.Cls.Disabled,
```

Comment:
shoule we disable CLS audit step if CLS is enabled

Reply:
You are right, it should be the other way around. Will fix it

Reply:
Fixed in 0e36826e3993c43002bdbe7ec6dbf9a530607d1e

Reply:
fixed.